README.md

```diff
@@ -1,7 +1,23 @@
 # Info Frontend
 
-This application exposes the user needs and metrics about content on GOV.UK.
+This application exposes the user needs and metrics about content on GOV.UK. It does this by fetching content details (including user needs) from [Content Store](https://github.com/alphagov/content-store).
```

Comment:
```suggestion
This application exposes the user needs and metrics about content on GOV.UK. It does this by fetching from [Content Store](https://github.com/alphagov/content-store).
```

Minor: I don't think we need to harp on about the needs ;-).

Reply:
lol but it's allllllllllways about user needs 😆 

Have removed some fluff from the sentence. 👍